_drafts/SC2-1-2-no-keyboard-trap-non-standard-navigation.md

```diff
@@ -0,0 +1,153 @@
+---
+name: No keyboard trap non-standard navigation
+group:
+- SC2-1-2-no-keyboard-trap-standard-navigation
+- SC2-1-2-no-keyboard-trap-non-standard-navigation (current)
+
+description: |
+  This rule checks if the user is advised on a method for non-standard keyboard navigation to navigate through focusable content on a web page without becoming trapped in any element.
+
+success_criterion: 
+- 2.1.2 # No Keyboard Trap
+
+test aspects:
+- DOM Tree
+
+authors:
+- Dagfinn Rømen
+- Geir Sindre Fossøy
+- Malin Øvrebø
+- Shadi Abou-Zahra
+- Carlos Duarte
+- Anne Thyme Nørregaard
+- Stein Erik Skotkjerra
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element on a web page that is [focusable][] and reachable through [sequential focus navigation](https://www.w3.org/TR/html/editing.html#sequential-focus-navigation) where focus cannot cycle to the browser UI by using [standard keyboard navigation][].
+
+### Expectation 1
+
+For each target element help information is [visible on the page][] and [exposed to assistive technologies][] or can be navigated to from within the keyboard trap.
```

Comment:
Add clarification: Does the information need to be visible while inside the keyboard trap, or could it have been displayed before, such as before you open a modal. Change  "can be navigated to" to "can be accessed"

Reply:
As a minimum, the information is visible while inside the keyboard trap. It could to be displayed before, as long as it is displayed when focus is trapped.